README.md

```diff
@@ -39,25 +39,9 @@ docker-compose up api
 docker-compose up nginx
 ```
 
-## Note about bcrypt
```

Comment:
Shoot, this change is not supposed to be here. I accidentally made this change when messing around with resetting my HEAD.